demo/nuxt.config.ts

```diff
@@ -129,6 +144,18 @@ export default {
         responseType: 'code',
         grantType: 'authorization_code',
         clientId: 'test-client'
+      },
+      oidcmock: {
+        scheme: 'openIDConnect',
+        responseType: 'code',
+        scope: ['openid', 'profile', 'offline_access'],
```

Comment:
Some of these options, like scope and response type, seem like good default OIDC options. What about setting those as defaults and removing them from the necessary config? Makes it a little easier to use.